src/frontend/src/components/SelectPicker.vue

```diff
@@ -56,7 +56,7 @@
 			@click="isOpen ? doClose() : doOpen($refs.focusOnClickOpen)"
 			@keydown.tab="doClose()/*focus shifts to next element, close menu*/"
 			@keydown.esc="doClose()"
-			@keydown.prevent.enter="$event.target.click()/*stop to prevent submitting any parent form*/"
+			@keydown.prevent.enter="clickTarget($event)/*stop to prevent submitting any parent form*/"
```

Comment:
I though enter always fires a 'click' event anyway?

Reply:
I'm not sure; I only moved this code into a method to prevent a TypeScript warning